client/layout/head/index.jsx

```diff
@@ -0,0 +1,52 @@
+/** @ssr-ready **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import Helmet from 'react-helmet';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import decodeEntities from 'lib/formatting/decode-entities';
+
+const debug = debugFactory( 'calypso:layout:head' );
+
+const Head = ( { title, isTitleFormatted, description, canonicalUrl, children } ) => (
+	<div>
+		<Helmet
+			title={ isTitleFormatted ? title : buildTitle( title ) }
```

Comment:
Client-side (i.e. controller-handled) Showcase passes a formatted title, which is built from sites data, etc., while server-side Showcase layout passes a title expected to be handled. To be honest, though, this handling only appends "— WordPress.com" to the original string, and we can hardcode that in `ThemesHead`. So yeah, it's maybe not `Head`'s responsability.